### ISSUE TITLE
"full stop code points"

### ISSUE THREAD

**Issue by annevk** [webspecs/url#13]
This is not an actual thing.

**Comment by rubys** [comment]
Unicode code point U+002E is defined by the Unicode standard as "FULL STOP": http://www.unicode.org/charts/PDF/U0000.pdf

**Comment by annevk** [comment]
Yes, but you use "full stop" (lowercase) and provide no reference. I don't think it follows from first principles what this text is on about.

**Comment by rubys** [comment]
Three occurrences of the words "full stop" in any case occur in the draft:

https://github.com/webspecs/url/blob/eec6b009ff9f4b97d04f49c33d7c53574fbaa191/url.bs#L856
https://github.com/webspecs/url/blob/eec6b009ff9f4b97d04f49c33d7c53574fbaa191/url.bs#L988
https://github.com/webspecs/url/blob/eec6b009ff9f4b97d04f49c33d7c53574fbaa191/url.bs#L1111

All are accompanied by the text <code>U+002D</code>.

**Comment by annevk** [comment]
No. The specific phrase I mentioned is at https://specs.webplatform.org/url/webspecs/develop/#ls32 and does not have this. What am I missing?

**Comment by annevk** [comment]
And why would U+002D be code? Is that a convention used elsewhere?

**Comment by annevk** [comment]
Also, if we use that convention why are you not applying that consistently?

**Comment by rubys** [comment]
> What am I missing?

My apologies.  I clearly missed that one.
